src/views/Login.java

```diff
@@ -234,28 +241,33 @@ public void mousePressed(MouseEvent e) {
 		header.setLayout(null);
 	}
 	
-	private void Login() {
-		 String Usuario= "admin";
-	     String Contraseña="admin";
-
-	        String contrase=new String (txtContrasena.getPassword());
-
-	        if(txtUsuario.getText().equals(Usuario) && contrase.equals(Contraseña)){
-	            MenuUsuario menu = new MenuUsuario();
-	            menu.setVisible(true);
-	            dispose();	 
-	        }else {
-	            JOptionPane.showMessageDialog(this, "Usuario o Contraseña no válidos");
-	        }
-	} 
-	 private void headerMousePressed(java.awt.event.MouseEvent evt) {
-	        xMouse = evt.getX();
-	        yMouse = evt.getY();
-	    }//GEN-LAST:event_headerMousePressed
+	private void executeLogin() {
+	
+	    String passw=new String(txtContrasena.getPassword());
+	
+	    if(dbLogin.loginUser(txtUsuario.getText(), passw)){
+	        MenuUsuario menu = new MenuUsuario();
+	        menu.setVisible(true);
+	        dispose();	 
+	    }else {
+	        JOptionPane.showMessageDialog(this, "Usuario o Contraseña no válidos");
```

Comment:
Let's try to use Unicode to avoid special characters on the code.

Here is a sample about how to use it.
http://trollinformatica.blogspot.com/2020/07/escribir-la-n-y-letra-con-tilde-dentro.html